examples/Server/Startup.cs

```diff
@@ -27,14 +29,28 @@ public class Startup
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddGrpc();
+            services
+                .AddGrpc(options =>
```

Comment:
Why is this a fluent API? It looks really weird

Reply:
It allows configuration to be customized per service. It's no different than the fluent API hanging off AddSignalR. Most people will set the global config in AddGrpc and not customize per service.

Do you have an alternative API?

Reply:
Defining an interceptor via an attributes would be the more user friendly way of configuring interceptors per service. See https://github.com/grpc/grpc-dotnet/pull/220#issuecomment-487243597

Reply:
My bad, I didn't notice the `IGrpcServerBuilder`. On first glance I assumed everything hangs off `IServiceCollection` which felt weird.

As an aside, I'm not really a fan that some `IServiceCollection` extension methods return builders whereas others return `IServiceCollection`. It feels inconsistent to me, but I just realized we do the same thing in MVC, SignalR, et al.